.changeset/chat-input-bar.md

```diff
@@ -0,0 +1,10 @@
+---
+'@lg-chat/input-bar': major
+---
+
+[LG-4121](https://jira.mongodb.org/browse/LG-4121): `InputBar` renders results menu in top layer using popover API. As a result, the following props are deprecated and removed:
```

Comment:
This one is codemod-able, right? Maybe we should mention the codemod in the changeset?